messagebus/src/vespa/messagebus/network/rpcnetwork.cpp

```diff
@@ -173,13 +178,13 @@ RPCNetwork::attach(INetworkOwner &owner)
     LOG_ASSERT(_owner == 0);
     _owner = &owner;
 
-    _sendV1.attach(*this);
-    _sendAdapters.insert(SendAdapterMap::value_type(vespalib::VersionSpecification(5), &_sendV1));
-    _sendAdapters.insert(SendAdapterMap::value_type(vespalib::VersionSpecification(6), &_sendV1));
+    _sendV1->attach(*this);
+    _sendV2->attach(*this);
+    _sendAdapters[vespalib::Version(5)] = _sendV1.get();
+    _sendAdapters[vespalib::Version(6, 142)] = _sendV2.get();
```

Comment:
Before final merge, versions have to be updated across languages to prevent handshake confusion. We're now at 6.147, but next release might be even higher